src/main/resources/db/migration/h2/V1_224_1__add_transcription_document_and_annotation_document_pk.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE transcription_document ADD CONSTRAINT transcription_document PRIMARY KEY (trd_id);
```

Comment:
This looks like it should be `transcription_document_pk` - similar to annotation_document_pk?

Reply:
ACCEPT